dockerHashing/src/main/scala/cromwell/docker/DockerInfoActor.scala

```diff
@@ -112,7 +113,7 @@ final class DockerInfoActor(
   /*
    * Sends back responses and adds to cache in case of success. Used as the sink for each registry stream.
    */
-  val streamSink = fs2.Sink[IO, (DockerInfoResponse, DockerInfoContext)] {
+  val streamSink: Pipe[IO, (DockerInfoResponse, DockerInfoContext), Unit] = _ evalMap {
```

Comment:
ici n'est pas une sink.

Reply:
[For the curious.](http://www.dereferer.org/?https%3A%2F%2Fgithub%2Ecom%2Ffunctional%2Dstreams%2Dfor%2Dscala%2Ffs2%2Fpull%2F1386)